src/main/java/gregtech/common/metatileentities/multi/electric/MetaTileEntityElectricBlastFurnace.java

```diff
@@ -75,8 +75,8 @@ protected void formStructure(PatternMatchContext context) {
         } else {
             this.blastFurnaceTemperature = CoilType.CUPRONICKEL.getCoilTemperature();
         }
-
-        this.blastFurnaceTemperature += 100 * Math.max(0, GTUtility.getTierByVoltage(getEnergyContainer().getInputVoltage()) - GTValues.MV);
+        //the subtracted tier gives the starting level (exclusive) of the +100K heat bonus
+        this.blastFurnaceTemperature += 100 * Math.max(0, GTUtility.getFloorTierByVoltage(getEnergyContainer().getInputVoltage()) - GTValues.LV);
```

Comment:
```suggestion
        this.blastFurnaceTemperature += 100 * Math.max(0, GTUtility.getFloorTierByVoltage(getEnergyContainer().getInputVoltage()) - GTValues.MV);
```

This bonus is intended to be given for every energy tier above MV, but not including it. Therefore, this should still subtract `GTValues.MV` so it does not apply for MV tier, but does at all voltages higher than it.